### DISCUSSION TITLE
Custom map source

### DISCUSSION THREAD

**Discussion by belugaming** [teslamate-org/teslamate#3997] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I am a user from China. I found a problem when using teslamate. The default map source cannot access map information due to GFW problems. Can we customize the map source or download offline maps and install them into teslamate?

### Relevant log output

```shell
0
```

### Type of installation

Docker

### Version

v1.28.5

**Comment by parkr** [comment]
Hello, did you search the issues and discussions before filing the bug report? I see many previous issues on this topic: https://github.com/teslamate-org/teslamate/issues/2989 https://github.com/teslamate-org/teslamate/issues/2956 https://github.com/teslamate-org/teslamate/issues/3023 https://github.com/teslamate-org/teslamate/issues/3097

**Comment by belugaming** [comment]
I checked these issues, but the solutions they proposed were to use another
mirror source, but the problem now is that the third-party mirror source
does not support ARM architecture processors.

Parker Moore ***@***.***>于2024年5月21日 周二12:04写道：

**Comment by belugaming** [comment]
Don’t you find that these issues were not successfully solved in the end?

ming beluga ***@***.***>于2024年5月21日 周二12:05写道：

**Comment by hipudding** [comment]
这个可以做，我这里有一个高德地图的[demo](https://github.com/hipudding/teslamate_fix_addrs/issues/7)，看是否能满足需求？

**Comment by cwanja** [comment]
Converting to discussion under **Ideas**. That is where we track feature requests. Issues are for current features that are classified as bugs. Thanks for the suggestion! We always encourage individuals to create a pull request as well.
